File: app/src/main/java/com/mahmoudrefaie/sekkawahda/ui/Registeration/Registeration.java
package com.mahmoudrefaie.sekkawahda.ui.Registeration;

import android.content.Intent;
import android.os.Bundle;
import android.view.MenuItem;
import android.view.View;
import android.widget.TextView;

import androidx.appcompat.app.AppCompatActivity;
import androidx.fragment.app.FragmentManager;

import com.mahmoudrefaie.sekkawahda.R;
import com.mahmoudrefaie.sekkawahda.ui.Login.Login;

import butterknife.BindView;
import butterknife.ButterKnife;

import androidx.appcompat.widget.Toolbar;

public class Registeration extends AppCompatActivity implements View.OnClickListener{

    @BindView(R.id.login)
    TextView login;
    @BindView(R.id.toolbar)
    Toolbar toolbar;
    @BindView(R.id.sign_up)
    TextView signUp;

    private FragmentRegisterA firstFragment;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_registeration);
        ButterKnife.bind(this);

        //BackArraw at ToolBar
        setSupportActionBar(toolbar);
        getSupportActionBar().setDisplayHomeAsUpEnabled(true);
        getSupportActionBar().setHomeAsUpIndicator(R.drawable.ic_arrow_back_blue_24dp);
        getSupportActionBar().setTitle("");

        //Open first Fragment that has username, email, password
        firstFragment = new FragmentRegisterA();
        FragmentManager fm = getSupportFragmentManager();
        fm.beginTransaction().add(R.id.fragment_register_layout , firstFragment).commit();

        login.setOnClickListener(this);
    }

    @Override
    public boolean onOptionsItemSelected(MenuItem item){
        switch (item.getItemId()){
            case android.R.id.home:
                onBackPressed();
                return true;
        }
        return super.onOptionsItemSelected(item);
    }

    @Override
    public void onClick(View view) {
        if(view.getId() == R.id.login) {
            Intent intent = new Intent(Registeration.this, Login.class);
            startActivity(intent);
            this.finish();
        }
    }
}

File: app/src/main/java/com/mahmoudrefaie/sekkawahda/ui/Registeration/Registeration.kt
package com.mahmoudrefaie.sekkawahda.ui.Registeration

import android.content.Intent
import android.os.Bundle
import android.view.MenuItem
import android.view.View
import android.widget.TextView
import androidx.appcompat.app.AppCompatActivity
import androidx.appcompat.widget.Toolbar
import butterknife.BindView
import butterknife.ButterKnife
import com.mahmoudrefaie.sekkawahda.R
import com.mahmoudrefaie.sekkawahda.databinding.ActivityLoginBinding
import com.mahmoudrefaie.sekkawahda.databinding.ActivityRegisterationBinding
import com.mahmoudrefaie.sekkawahda.ui.Login.Login

class Registeration : AppCompatActivity(), View.OnClickListener {

    private lateinit var binding: ActivityRegisterationBinding

//    @JvmField
//    @BindView(R.id.login)
//    var login: TextView? = null
//
//    @JvmField
//    @BindView(R.id.toolbar)
//    var toolbar: Toolbar? = null

    private var firstFragment: FragmentRegisterA? = null

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        binding = ActivityRegisterationBinding.inflate(getLayoutInflater());
        setContentView(binding.root)
        //setContentView(R.layout.activity_registeration)
        //ButterKnife.bind(this)

        //BackArraw at ToolBar
        setSupportActionBar(binding.toolbar)
        supportActionBar!!.setDisplayHomeAsUpEnabled(true)
        supportActionBar!!.setHomeAsUpIndicator(R.drawable.ic_arrow_back_blue_24dp)
        supportActionBar!!.title = ""

        //Open first Fragment that has username, email, password
        firstFragment = FragmentRegisterA()
        val fm = supportFragmentManager
        fm.beginTransaction()
                .add(R.id.fragment_register_layout, firstFragment!!)
                .addToBackStack("first_fragment")
                .commit()

        binding.login!!.setOnClickListener(this)
    }

    override fun onOptionsItemSelected(item: MenuItem): Boolean {
        when (item.itemId) {
            android.R.id.home -> {
                Activity.onBackPressed()()
                return true
            }
        }
        return super.onOptionsItemSelected(item)
    }

    override fun onClick(view: View) {
        if (view.id == R.id.login) {
            val intent = Intent(this@Registeration, Login::class.java)
            startActivity(intent)
            finish()
        }
    }
}